pynest/nest/lib/hl_api_types.py

```diff
@@ -439,7 +421,7 @@ def get(self, *params, **kwargs):
                 result = {key: [val] for key, val in result.items()}
             result = pandas.DataFrame(result, index=index)
         elif output == 'json':
-            result = _serialize(result)
+            result = serializable(result)
```

Comment:
I think that instead of using `serializable` here, you should use the `to_JSON` function (also in `hl_api_helper`). `to_JSON` calls `serializable` behind the scene. In NEST#1123 it was decided that we should use the `to_JSON` function for `Get_Status` and `Get_Defaults` because when you say `output=JSON`, you would expect to get something back that is in JSON format. And I think we should be consistent. You might have to change some tests in `test_get_set.py`, but I am not sure.